### ISSUE TITLE
Amazon Voice Focus plays out silence in Chrome Canary 95

### ISSUE THREAD

**Issue by richnew10** [aws/amazon-chime-sdk-js#1588]
### What happened and what did you expect to happen?

When Voice Focus is enabled in the demo app, the mic indicator doesn't move.

Works fine in 93.

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

1. Open the demo app in Chrome 95.
2. Enable Web Audio and click next.
3. Speak: see mic indicator move.
4. Click the Voice Focus checkbox.
5. Speak.

Expected: mic indicator moves.

Actual: it does not.

### Amazon Chime SDK for JavaScript version

current

### What browsers are you seeing the problem on?

Chrome

### Browser version

95

### Browser console logs

```
[DEMO] [DEMO] Amazon Voice Focus setting toggled to true
[DEMO] VF supported: true
[DEMO] VF enabled: true
[DEMO] Selecting audio input VoiceFocusTransformDevice
[INFO] SDK - API/DefaultAudioVideoFacade/627fcddd-6b0a-4b0a-9478-2f69af870706/c0f987f1-cece-5ce5-b5bc-da395d2b5f29/chooseAudioInputDevice {"device":"default","voiceFocus":{"processorURL":"https://static.sdkassets.chime.aws/processors/worklet-inline-processor-v1.js","nodeOptions":{"processor":"voicefocus-inline-processor","worker":{},"audioBufferURL":"https://static.sdkassets.chime.aws/wasm/audio_buffer-v1_simd.wasm","resamplerURL":"https://static.sdkassets.chime.aws/wasm/resampler-v1_simd.wasm","fetchBehavior":{"paths":{"processors":"https://static.sdkassets.chime.aws/processors/","workers":"https://static.sdkassets.chime.aws/workers/","wasm":"https://static.sdkassets.chime.aws/wasm/","models":"https://static.sdkassets.chime.aws/wasm/"},"escapedQueryString":"sdk=2.16.1&ua=chrome-95&assetGroup=sdk-2.16"},"modelURL":"https://static.sdkassets.chime.aws/static/voicefocus-default-c100-v1_simd-e7bad7b961a128cf.wasm","delegate":{"observers":{}},"logger":{"base":{"name":"SDK","level":0}}}}},"delegate":{"observers":{}},"failed":false,"browserBehavior":{"browser":{"name":"chrome","version":"95.0.4638","os":"Mac OS","type":"browser"},"browserSupport":{"chrome":78,"edge-chromium":79,"electron":7,"firefox":60,"ios":12,"safari":12,"opera":66,"samsung":12,"crios":86,"fxios":23,"ios-webview":605,"chromium-webview":92},"browserName":{"chrome":"Google Chrome","edge-chromium":"Microsoft Edge","electron":"Electron","firefox":"Mozilla Firefox","ios":"Safari iOS","safari":"Safari","opera":"Opera","samsung":"Samsung Internet","crios":"Chrome iOS","fxios":"Firefox iOS","ios-webview":"WKWebView iOS","chromium-webview":"Chrome WebView"},"chromeLike":["chrome","edge-chromium","chromium-webview","opera","samsung"],"webkitBrowsers":["crios","fxios","safari","ios","ios-webview"],"enableUnifiedPlanForChromiumBasedBrowsers":false,"recreateAudioContextIfNeeded":false}}
[INFO] SDK - Choosing transform input device [object Object]
[DEBUG] SDK - VoiceFocusInlineNode: https://static.sdkassets.chime.aws/static/voicefocus-default-c100-v1_simd-e7bad7b961a128cf.wasm
//static.sdkassets.chime.aws/static/voicefocus-default-c100-v1_simd-e7bad7b961a128cf.wasm already loaded!
[INFO] SDK - requesting new audio device with constraint {"audio":{"echoCancellation":true,"googEchoCancellation":true,"googEchoCancellation2":true,"googAutoGainControl":true,"googAutoGainControl2":true,"googNoiseSuppression":false,"googNoiseSuppression2":false,"googHighpassFilter":false,"noiseSuppression":false,"autoGainControl":true,"deviceId":{"exact":"default"},"sampleRate":{"ideal":48000},"sampleSize":{"ideal":16},"channelCount":{"ideal":1}}}
[INFO] SDK - got audio device for constraints {"audio":{"echoCancellation":true,"googEchoCancellation":true,"googEchoCancellation2":true,"googAutoGainControl":true,"googAutoGainControl2":true,"googNoiseSuppression":false,"googNoiseSuppression2":false,"googHighpassFilter":false,"noiseSuppression":false,"autoGainControl":true,"deviceId":{"exact":"default"},"sampleRate":{"ideal":48000},"sampleSize":{"ideal":16},"channelCount":{"ideal":1}}}
[DEMO] Updating Amazon Voice Focus display state: true
[DEMO] Starting audio preview.
6 2021-09-10T19:27:06.987Z [INFO] SDK - API/DefaultAudioVideoFacade/627fcddd-6b0a-4b0a-9478-2f69af870706/c0f987f1-cece-5ce5-b5bc-da395d2b5f29/createAnalyserNodeForAudioInput
[DEMO] Mute state: device default is not muted
```

**Comment by richnew10** [comment]
Debugger tells me that the worklet is never getting out of the 'ready' state; something in `makeReady` or that calls it (e.g., message handling) is failing. I'm not seeing any logging from the processor in 95, so it's either not getting messages, or calling `console.log` is throwing.

**Comment by richnew10** [comment]
The 95 changelog includes:

https://bugs.chromium.org/p/chromium/issues/detail?id=1213818

https://chromium.googlesource.com/chromium/src/+/108cf1b27385c6d25f45f6526f89a9ae8ea88f23

which could be relevant.

**Comment by wanchenjade** [comment]
Root cause is worker postMessage is not working correctly in sending WebAssembly module https://code.amazon.com/packages/ChimeNoiseSuppressionWeb/blobs/e1c6573d3062e992acce4346a20232ba9614343b/--/web/workers/worker-v1.js#L355

From Chrome 95 on webassembly module is not allowed to be put on postMessage. We need to find new way to pass in webassembly module. 
https://developer.chrome.com/blog/wasm-module-sharing-restricted-to-same-origin/
https://www.chromestatus.com/feature/5650158039597056

**Comment by richnew10** [comment]
Fixed in #1625.
